src/Silverback.Integration/Silverback.Integration.csproj

```diff
@@ -36,5 +36,7 @@ This package contains the message broker and connectors abstractions.</Descripti
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Silverback.Core" Version="$(BaseVersion)" />
     <PackageReference Include="System.Memory" Version="4.5.2" />
+    <PackageReference Include="System.Memory" Version="4.5.3" />
```

Comment:
Something is still broken for me. It doens't compile on azure devops nor on my mac.

Reply:
Jup, will fix this after we got the implementation details correct.